### PR TITLE
Feature/refactor sql scripts

### PR DESCRIPTION
**Pull request by beneditatan**
#### Reviewed by:
- [x] @hixi
- [x] @das-g

### CODE REVIEW

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/address/000_setup-drop_and_recreate_table.sql [hunk 1]
```diff
@@ -22,3 +22,6 @@ CREATE TABLE osmaxx.address_p(
 	city text,
     country text
 );
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/address/000_setup-drop_and_recreate_table.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/address/010_address.sql [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		when "addr:place" is not null then "addr:place"
+		else NULL
+    end  as street,
+	"addr:housenumber" as housenumber,
+	"addr:postcode" as postcode,
+	"addr:place" as city,
+    "addr:country" as country
+  FROM osm_polygon
+where building !='entrance' and entrance is null and ("addr:street" !='' OR "addr:housenumber"!='' OR "addr:place"!='');
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/address/010_address.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/address/020_entrance.sql [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        when "addr:place" is not null then "addr:place"
+        else NULL
+    end  as street,
+	"addr:housenumber" as housenumber,
+	"addr:postcode" as postcode,
+	"addr:place" as city,
+    "addr:country" as country
+  FROM osm_point
+  where ("addr:street" !='' OR "addr:housenumber"!='' OR "addr:place"!='');
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/address/020_entrance.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/address/030_interpolation.sql [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+	temp_tbl.addr_street as street,
+	temp_tbl.housenr as housenumber,
+	osm_line."addr:postcode" as postcode,
+	osm_line."addr:place" as city,
+    osm_line."addr:country" as country
+ FROM temp_tbl 
+ INNER JOIN osm_line
+ ON temp_tbl.line_id=osm_line.osm_id;
+DROP TABLE temp_tbl;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/address/030_interpolation.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/000_setup-drop_and_recreate_table_adminarea.sql [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+	name text, 
+	name_en text, 
+	name_fr text, 
+	name_es text, 
+	name_de text,
+	int_name text,
+	label text,
+	tags text
+);
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/000_setup-drop_and_recreate_table_adminarea.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/010_adminarea.sql [hunk 6]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/010_adminarea.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/010_adminarea.sql [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+	"name:en" as name_en, 
+	"name:fr" as name_fr, 
+	"name:es" as name_es, 
+	"name:de" as name_de, 
+	int_name as name_int, 
+	transliterate(name) as label,
+	cast(tags as text) as tags
+  FROM osm_polygon
+  WHERE boundary='administrative' or boundary='national_park' or boundary='protected_area';
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/010_adminarea.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/020_setup-drop_and_recreate_table_boundary.sql [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+	type text, 
+	name text,
+	name_en text, 
+	name_fr text, 
+	name_es text, 
+	name_de text,
+	int_name text,
+	label text,
+	tags text
+);
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/020_setup-drop_and_recreate_table_boundary.sql]: Please add a newline.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/030_boundary.sql [hunk 9]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/030_boundary.sql]: Please remove the leading newline.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/030_boundary.sql [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+	"name:en" as name_en, 
+	"name:fr" as name_fr, 
+	"name:es" as name_es, 
+	"name:de" as name_de, 
+	int_name as name_int, 
+	transliterate(name) as label,
+	cast(tags as text) as tags
+  FROM osm_line
+  WHERE boundary='administrative' or boundary='national_park' or boundary='protected_area';
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/030_boundary.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/building/000_setup-drop_and_recreate_table_building.sql [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+	name_en text, 
+	name_fr text, 
+	name_es text, 
+	name_de text,
+	int_name text,
+	label text,
+	tags text,
+	height float
+);
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/building/000_setup-drop_and_recreate_table_building.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/building/010_building.sql [hunk 12]
```diff
@@ -40,3 +19,6 @@ INSERT INTO osmaxx.building_a
   FROM osm_polygon
   WHERE building is not null;
 
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/building/010_building.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/000_view_address.sql [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+	name, 
+	label,
+	tags,
+	street,
+	housenumber,
+	postcode,
+	city,
+	country
+FROM osmaxx.address_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/000_view_address.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/001_view_adminarea.sql [hunk 14]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/001_view_adminarea.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/001_view_adminarea.sql [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.adminarea_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/001_view_adminarea.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/002_view_boundary.sql [hunk 16]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/002_view_boundary.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/002_view_boundary.sql [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.boundary_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/002_view_boundary.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/003_view_building.sql [hunk 18]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/003_view_building.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/003_view_building.sql [hunk 19]
```diff
@@ -0,0 +1,15 @@
...
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags,
+	height
+FROM osmaxx.building_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/003_view_building.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/004_view_geoname_l.sql [hunk 20]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/004_view_geoname_l.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/004_view_geoname_l.sql [hunk 21]
```diff
@@ -0,0 +1,15 @@
...
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags,
+	population
+FROM osmaxx.geoname_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/004_view_geoname_l.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/005_view_geoname_p.sql [hunk 22]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/005_view_geoname_p.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/005_view_geoname_p.sql [hunk 23]
```diff
@@ -0,0 +1,16 @@
...
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags,
+	population
+FROM osmaxx.geoname_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/005_view_geoname_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/006_view_landuse_a.sql [hunk 24]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/006_view_landuse_a.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/006_view_landuse_a.sql [hunk 25]
```diff
@@ -0,0 +1,16 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.landuse_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/006_view_landuse_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/007_view_military_a.sql [hunk 26]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/007_view_military_a.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/007_view_military_a.sql [hunk 27]
```diff
@@ -0,0 +1,15 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.military_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/007_view_military_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/008_view_military_p.sql [hunk 28]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/008_view_military_p.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/008_view_military_p.sql [hunk 29]
```diff
@@ -0,0 +1,14 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.military_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/008_view_military_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/009_view_misc_l.sql [hunk 30]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/009_view_misc_l.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/009_view_misc_l.sql [hunk 31]
```diff
@@ -0,0 +1,16 @@
...
+	lastchange, 
+	geomtype,
+	geom, 
+	aggtype,
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.misc_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/009_view_misc_l.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/010_view_natural_a.sql [hunk 32]
```diff
@@ -0,0 +1,13 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.natural_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/010_view_natural_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/011_view_natural_p.sql [hunk 33]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/011_view_natural_p.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/011_view_natural_p.sql [hunk 34]
```diff
@@ -0,0 +1,14 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.natural_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/011_view_natural_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/012_view_nonop_l.sql [hunk 35]
```diff
@@ -0,0 +1,16 @@
...
+	geom, 
+	type, 
+	name, 
+	label,
+	tags,
+	ref,
+	z_order,
+	status
+FROM osmaxx.nonop_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/012_view_nonop_l.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/013_view_pow_a.sql [hunk 36]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/013_view_pow_a.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/013_view_pow_a.sql [hunk 37]
```diff
@@ -0,0 +1,21 @@
...
+	label,
+	tags,
+	website,
+	wikipedia,
+	phone,
+	contact_phone,
+	opening_hours,
+	"access"
+FROM osmaxx.pow_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/013_view_pow_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/014_view_pow_p.sql [hunk 38]
```diff
@@ -0,0 +1,20 @@
...
+	label,
+	tags,
+	website,
+	wikipedia,
+	phone,
+	contact_phone,
+	opening_hours,
+	"access"
+FROM osmaxx.pow_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/014_view_pow_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/015_view_poi_a.sql [hunk 39]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/015_view_poi_a.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/015_view_poi_a.sql [hunk 40]
```diff
@@ -0,0 +1,26 @@
...
+	wikipedia,
+	phone,
+	contact_phone,
+	opening_hours,
+	cuisine,
+	"access",
+	brand,
+	tower_type
+FROM osmaxx.poi_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/015_view_poi_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/016_view_poi_p.sql [hunk 41]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/016_view_poi_p.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/016_view_poi_p.sql [hunk 42]
```diff
@@ -0,0 +1,24 @@
...
+	wikipedia,
+	phone,
+	contact_phone,
+	opening_hours,
+	cuisine,
+	"access",
+	brand,
+	tower_type
+FROM osmaxx.poi_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/016_view_poi_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/017_view_railway_l.sql [hunk 43]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/017_view_railway_l.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/017_view_railway_l.sql [hunk 44]
```diff
@@ -0,0 +1,20 @@
...
+	name, 
+	label,
+	tags,
+	z_order,
+	bridge,
+	tunnel,
+	voltage,
+	frequency
+FROM osmaxx.railway_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/017_view_railway_l.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/018_view_road_l.sql [hunk 45]
```diff
@@ -0,0 +1,19 @@
...
+	name, 
+	label,
+	tags,
+	ref,
+	oneway,
+	z_order,
+	bridge,
+	tunnel
+FROM osmaxx.road_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/018_view_road_l.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/019_view_route_l.sql [hunk 46]
```diff
@@ -0,0 +1,15 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.route_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/019_view_route_l.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/019_view_route_l.sql [hunk 47]
```diff
@@ -0,0 +1,15 @@
...
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/019_view_route_l.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/020_view_traffic_a.sql [hunk 48]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/020_view_traffic_a.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/020_view_traffic_a.sql [hunk 49]
```diff
@@ -0,0 +1,16 @@
...
+	geomtype,
+	geom, 
+	aggtype,
+	type, 
+	name, 
+	label,
+	tags,
+	"access"
+FROM osmaxx.traffic_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/020_view_traffic_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/021_view_traffic_p.sql [hunk 50]
```diff
@@ -0,0 +1,15 @@
...
+	geomtype,
+	geom, 
+	aggtype,
+	type, 
+	name, 
+	label,
+	tags,
+	"access"
+FROM osmaxx.traffic_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/021_view_traffic_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/022_view_transport_a.sql [hunk 51]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/022_view_transport_a.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/022_view_transport_a.sql [hunk 52]
```diff
@@ -0,0 +1,17 @@
...
+	lastchange, 
+	geomtype,
+	geom,
+	aggtype,
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.transport_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/022_view_transport_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/023_view_transport_p.sql [hunk 53]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/023_view_transport_p.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/023_view_transport_p.sql [hunk 54]
```diff
@@ -0,0 +1,15 @@
...
+	lastchange, 
+	geomtype,
+	geom, 
+	aggtype,
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.transport_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/023_view_transport_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/024_view_utility_a.sql [hunk 55]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/024_view_utility_a.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/024_view_utility_a.sql [hunk 56]
```diff
@@ -0,0 +1,15 @@
...
+	lastchange, 
+	geomtype,
+	geom,
+	aggtype,
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.utility_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/024_view_utility_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/025_view_utility_p.sql [hunk 57]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/025_view_utility_p.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/025_view_utility_p.sql [hunk 58]
```diff
@@ -0,0 +1,16 @@
...
+	lastchange, 
+	geomtype,
+	geom,
+	aggtype,
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.utility_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/025_view_utility_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/026_view_utility_l.sql [hunk 59]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/026_view_utility_l.sql]: Please remove the leading newline.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/026_view_utility_l.sql [hunk 60]
```diff
@@ -0,0 +1,17 @@
...
+	geom, 
+	type, 
+	name, 
+	label,
+	tags,
+	operator,
+	voltage,
+	frequency
+FROM osmaxx.utility_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/026_view_utility_l.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/027_view_water_a.sql [hunk 61]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/027_view_water_a.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/027_view_water_a.sql [hunk 62]
```diff
@@ -0,0 +1,14 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.water_a;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/027_view_water_a.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/028_view_water_p.sql [hunk 63]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/028_view_water_p.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/028_view_water_p.sql [hunk 64]
```diff
@@ -0,0 +1,17 @@
...
+	osm_id, 
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags
+FROM osmaxx.water_p;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/028_view_water_p.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/029_view_water_l.sql [hunk 65]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/029_view_water_l.sql]: Please remove the leading newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/029_view_water_l.sql [hunk 66]
```diff
@@ -0,0 +1,17 @@
...
+	lastchange, 
+	geomtype,
+	geom, 
+	type, 
+	name, 
+	label,
+	tags,
+	width
+FROM osmaxx.water_l;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/create_view/029_view_water_l.sql]: Please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/route/010_route.sql [hunk 67]
```diff
@@ -40,3 +21,7 @@ INSERT INTO osmaxx.route_l
 	FROM osm_line
 	WHERE route is not null;
 
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/route/010_route.sql]: please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/utility/040_utility_p_power.sql [hunk 68]
```diff
@@ -0,0 +1,96 @@
...
+	"name:en" as name_en, 
+	"name:fr" as name_fr, 
+	"name:es" as name_es, 
+	"name:de" as name_de, 
+	int_name as name_int, 
+	transliterate(name) as label,
+	cast(tags as text) as tags
+  FROM osm_polygon
+  WHERE "power" is not null or power_source is not null;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/utility/040_utility_p_power.sql]: please remove the trailing newlines.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/poi/190_poi_p_sport.sql [hunk 69]
```diff
@@ -0,0 +1,79 @@
...
+	phone as phone,
+	"contact:phone" as contact_phone,
+	opening_hours as opening_hours,
+	cuisine as cuisine,
+	"access" as "access",
+	brand as brand,
+	"tower:type" as tower_type
+  FROM osm_polygon
+  WHERE sport is not null;
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/poi/190_poi_p_sport.sql]: please add a newline.

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/route/000_setup-drop_and_recreate_table_route.sql [hunk 70]
```diff
@@ -0,0 +1,22 @@
...
+	name text, 
+	name_en text, 
+	name_fr text, 
+	name_es text, 
+	name_de text,
+	int_name text,
+	label text,
+	tags text
+);
+
```
Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/route/000_setup-drop_and_recreate_table_route.sql]: please remove the trailing newlines.

**Comment by hixi** [comment]
I'm very pleased with this, as soon as all newline remarks are done :-)

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/building/010_building.sql [hunk 71]
```diff
@@ -39,4 +18,4 @@ INSERT INTO osmaxx.building_a
 	cast(nullif(height,'') as float) as height
   FROM osm_polygon
   WHERE building is not null;
-
+
```
Comment (das-g) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/building/010_building.sql]: gratuitous space after final newline

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/drop_and_recreate/drop_and_recreate.sql [hunk 72]
```diff
@@ -5,3 +5,5 @@ DROP SCHEMA IF EXISTS osmaxx;
 -- CREATE SCHEMAS
 CREATE SCHEMA osmaxx;
 CREATE SCHEMA view_osmaxx;
+
+
```
Comment (das-g) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/drop_and_recreate/drop_and_recreate.sql]: Why was this file changed at all?

Comment (hixi) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/drop_and_recreate/drop_and_recreate.sql]: I guess this just happened while trying to address all newline issues, maybe @beneditatan knows more? ;-)

File: conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/030_boundary.sql (hunk 9 above)
Comment (das-g) [inline conversion_service/converters/gis_converter/bootstrap/sql/filter/adminarea_boundary/030_boundary.sql]: :white_check_mark: Has been fixed with 19d9863a4139412bc3bdfdf733c41379e61914cf. GitHub just found another empty line to attach this comment to in later versions.
